### ISSUE TITLE
Bug on multiline assignments

### ISSUE THREAD

**Issue by JoaoFelipe** [gems-uff/noworkflow#109]
noWorkflow captures variable values in the line immediately after the variable definition. 
This causes a bug on multiline assignments:

``` python
def double(x):
    return 2 * x

a = 1
b = 2
a = double(
                  b)
c = 3
```

In this example, noWorkflow will capture 2 variables "a" with the value 1, but the second one should have the value 4. It occurs because we try to capture the value of the second "a", when the assignment is not complete.

**Comment by JoaoFelipe** [comment]
This bug will be fixed by #84

**Comment by JoaoFelipe** [comment]
Transformer branch:
now dataflow -m simulation --value-length 50 -e 0

![i109](https://user-images.githubusercontent.com/327789/26948330-f274fd5c-4c63-11e7-9f25-41fecabf7bec.png)
